__tests__/unit/_publish/set-auth-token.test.js

```diff
@@ -66,8 +66,8 @@ describe('Set the auth token inside the correct .npmrc file', () => {
 		mockos({'homedir': 'home/user-c'});
 		expect.assertions(1);
 		await expect(
-			setAuthToken('path/to/auth-package', null)
-		).rejects.toBeInstanceOf(Error);
+			setAuthToken('path/to/auth-package')
+		).rejects.toThrowError(new Error('Authentication token already set for https://registry.npmjs.org/'));
```

Comment:
when testing Error messages I have started putting the strings in a new module and consuming it between main and test code, but that can wait ;)

Reply:
yep, I changed this to test on the actual message as I found a test was throwing but not in the way I expected! It was a bit of code smell that the messages had to exist in two different places, so this would be a nice solution. Will add to the list!

Reply:
As an aside, I also found this a good read: https://www.joyent.com/node-js/production/design/errors

Pertinent here is the section way down the page beginning "Use Error objects (or subclasses) for all errors" for constructing fancier Error objects. Have yet to try this out though.